ios/AppLovinMAX.m

```diff
@@ -241,6 +241,13 @@ - (instancetype)init
     return @([ALPrivacySettings hasUserConsent]);
 }
 
+RCT_EXPORT_METHOD(showConsentDialog:(RCTResponseSenderBlock)callback)
```

Comment:
Can we declare this before `getConsentDialogState()` like how we have it in Unity

Reply:
I changed the order for all - ios, java, and index.js.